apps/react-templates/.gitlab-ci.yml

```diff
@@ -1,18 +1,15 @@
-.tmpl:master_deploy_rules: &tmpl_master_deploy_rules
+deploy:ses:master:
   rules:
     - if: $CI_COMMIT_BRANCH == 'master'
-  variables: &tmpl_master_deploy_variables
-    SLS_STAGE: production
-
-deploy:ses:master:
-  <<: *tmpl_master_deploy_rules
-  dependencies:
-    - build
   script:
     - yarn node apps/react-templates/lib/deploy.js
   stage: deploy
+  dependencies:
+    - setup
+    - build
   needs:
+    - setup
     - build
   variables:
-    <<: *tmpl_master_deploy_variables
-    ORIGIN: https://$BASE_URL
+    SLS_STAGE: production
+    APP_ORIGIN: https://$APP_ORIGIN
```

Comment:
This is very confusing, either it's with a protocol, then it's an `origin`, or it's without a protocol, then it's `host`. We shouldn't have `origin` sometimes include protocol and sometimes not. In this case, it seems like the reference should be `$APP_HOST`.

Reply:
The script exports just hostnames and they are used as is or with a protocol when used as origins.